node_modules/gh-orgunit/tests/test-series.js

```diff
@@ -98,18 +98,34 @@ describe('Organisational units - Series', function() {
                 SeriesTestUtil.assertCreateSerie(global.tests.admins.cam2013.client, 'Test serie', null, function(serie) {
 
                     // Invalid organisational unit id
-                    OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, 'not a number', [serie.id], 400, function() {
-                        OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, -1, [serie.id], 404, function() {
+                    OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, 'not a number', [serie.id], null, 400, function() {
+                        OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, -1, [serie.id], null, 404, function() {
 
                             // Empty series
-                            OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [], 400, function() {
+                            OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [], null, 400, function() {
 
                                 // Invalid serie id
-                                OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, 'not a number', 400, function() {
-                                    OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, -1, 404, function() {
-                                        OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [serie.id, 'not a number'], 400, function() {
-                                            OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [serie.id, -1], 404, function() {
-                                                return callback();
+                                OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, 'not a number', null, 400, function() {
+                                    OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, -1, null, 404, function() {
+                                        OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [serie.id, 'not a number'], null, 400, function() {
+                                            OrgUnitTestUtil.assertAddOrgUnitSeriesFails(global.tests.admins.cam2013.client, orgUnit.id, [serie.id, -1], null, 404, function() {
+
+                                                // Invalid from organisational unit id
```

Comment:
Can we come up with a better description for this parameter? I keep misreading "from organisational unit". Just making it "borrowed from organisational unit" would help.